pages/_/marketing-candidate/download.js

```diff
@@ -48,17 +50,18 @@ const STYLES_H3 = css`
   }
 `;
 
-const STYLES_TEXT_BLOCK = css`
+const STYLES_BLOCK = css`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   max-width: 56vw;
   width: 100%;
   align-self: center;
+  margin-bottom: 16px;
 
   @media (max-width: ${Constants.sizes.mobile}px) {
     width: 100%;
-    right: 24px;
+    max-width: 100vw;
```

Comment:
What does this do?

Reply:
to overite the `max-width: 56vw` in desktop